tensorflow/python/data/kernel_tests/iterator_test.py

```diff
@@ -843,7 +913,8 @@ def testIteratorEagerIteration(self, execution_mode):
         self.assertEqual(val, foo.numpy())
         val += 1
 
-  @test_util.run_v2_only
+  @combinations.generate(
+      combinations.combine(tf_api_version=2, mode="eager"))
```

Comment:
I'm not sure if my change here is correct. 
* IIUC, `@test_util.run_v2_only` will run the test in both eager and graph mode, but only the eager tensor has `numpy()`, which is used at [self.assertEqual(queue.size().numpy(), 2)](https://github.com/tensorflow/tensorflow/pull/32599/files#diff-edaa5071d8e3b55304a0797163bb7cbfL861). 
* The change here will only run the test in the eager mode.

Reply:
`@test_util.run_v2_only` runs V2 * eager
`@test_util.run_v1_only` runs V1 * graph
`@test_util.deprecated_graph_mode_only` runs (V1 + V2) * graph

Reply:
Gotcha, thanks!